lib/core/src/send_swap.rs

```diff
@@ -359,25 +359,24 @@ impl SendSwapStateHandler {
     }
 
     async fn refund(&self, swap: &SendSwap) -> Result<String, PaymentError> {
-        let amount_sat = get_invoice_amount!(swap.invoice);
         let output_address = self.onchain_wallet.next_unused_address().await?.to_string();
 
-        let fee = self
-            .onchain_wallet
-            .build_tx(None, &output_address, amount_sat)
-            .await?
-            .all_fees()
-            .values()
-            .sum();
+        let cooperative_refund_tx_fees_sat =
+            utils::estimate_refund_fees(swap, &self.config, &output_address, false)?;
```

Comment:
Currently, setting the `is_lowball` check to false seems to be the only option that works on mainnet (thus defaulting to 0.1 sat/vbyte). Are we sure we can broadcast lowball through boltz when attempting refunds (using 0.01 causes a min_relay_fee not met error, as it should be >129 but is actually 14).

Reply:
@hydra-yse are you sure it fails for the cooperative case with lowball fees?  I am asking because the cooperative case seems to be using boltz api which supports lowball for the cooperative claim.

Reply:
I see there are some conditions to the lowball in the boltz-backend
> // Only allow lowball when Reverse Swaps are being claimed or Submarine Swaps being refunded
   // and no Submarine Swap lockup happens
   // That prevents transactions that we could accept 0-conf for from being broadcast through our API
    const isSwapRelated = swapsFunded.length === 0 && swapsSpent.length > 0;

Reply:
@dangeross do you understand how could be a refundable submarine swap without lockup? isn't the point that the refund tx spends the lockup?

Reply:
True, the comment doesn't really make sense. From looking at it it allows lowball on the lockup, no lowball on the refund
https://github.com/BoltzExchange/boltz-backend/blob/5d2cab089771cfb96be23349cfc3432c7b174e25/lib/service/Service.ts#L707-L722

Reply:
Looking further at the code here: https://github.com/BoltzExchange/boltz-backend/blob/5d2cab089771cfb96be23349cfc3432c7b174e25/lib/service/TransactionFetcher.ts#L184

It seems that boltz means by the comment that they don't want to allow broadcasting lockups via their API.
It seems that they demand that the broadcasted transaction outputs are not related to any lockup script.
Which seems meet our criteria for the refund as they also validates it spends a current lockup script.

Reply:
@hydra-yse I see that `util::estimate_refund_fees` is hardcoding `is_cooperative = None` when singing the refund tx. Could it be that's why it fails to broadcast with the lowball feerate? Have you tried setting the `is_cooperative` arg of `sign_refund` to `Some` if `is_lowball` is true?